package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "about-window",
-  "version": "1.11.0",
+  "version": "1.12.0",
```

Comment:
Could you revert this change? Let me create another commit to bump version to create a tag commit after merging this PR.

Reply:
Will do!

Reply:
Done!